### PR TITLE
Document HowTo use dapr instead of daprd in IntelliJ.

### PR DESCRIPTION
**Pull request by artursouza**
## Description

Instructions on how to use IntelliJ with `dapr` instead of `daprd`.
Instructions on how to create reusable `dapr` commands in IntelliJ via use of $Prompt$ for user input.

## Issue reference

_Please reference the issue this PR will close: #684

### CODE REVIEW

File: howto/intellij-debugging-daprd/README.md [hunk 1]
```diff
@@ -39,6 +39,37 @@ Create or edit the file in `$HOME/.IdeaIC2019.3/config/tools/External\ Tools.xml
 </toolSet>
 ```
 
+Optionally, you may also create a new entry for a sidecar tool that can be reused accross many projects:
+
+```xml
+<toolSet name="External Tools">
+  ...
+  <!-- 1. Reusable entry for apps with app port. -->
+  <tool name="daprd with app-port" description="Dapr sidecar" showInMainMenu="false" showInEditor="false" showInProject="false" showInSearchPopup="false" disabled="false" useConsole="true" showConsoleOnStdOut="true" showConsoleOnStdErr="true" synchronizeAfterRun="true">
```
Comment (pruthvidhodda) [inline howto/intellij-debugging-daprd/README.md:48]: daprd --> dapr

File: howto/intellij-debugging-daprd/README.md [hunk 2]
```diff
@@ -39,6 +39,37 @@ Create or edit the file in `$HOME/.IdeaIC2019.3/config/tools/External\ Tools.xml
...
+      <!-- 2. For Linux or MacOS use: /usr/bin/dapr -->
+      <option name="COMMAND" value="c:\dapr\dapr.exe" />
+      <!-- 3. Prompts user 4 times (in order): app id, app port, Dapr's http port, Dapr's grpc port. -->
+      <option name="PARAMETERS" value="run --app-id $Prompt$ --app-port $Prompt$ --dapr-http-port $Prompt$ --dapr-grpc-port $Prompt$" />
+      <!-- 4. Use the folder where the `components` folder is located -->
+      <option name="WORKING_DIRECTORY" value="$ProjectFileDir$" />
+    </exec>
+  </tool>
+  <!-- 1. Reusable entry for apps without app port. -->
+  <tool name="daprd without app-port" description="Dapr sidecar" showInMainMenu="false" showInEditor="false" showInProject="false" showInSearchPopup="false" disabled="false" useConsole="true" showConsoleOnStdOut="true" showConsoleOnStdErr="true" synchronizeAfterRun="true">
```
Comment (pruthvidhodda) [inline howto/intellij-debugging-daprd/README.md:59]: daprd --> dapr

**Comment by artursouza** [comment]
One catch with the renaming is to update the corresponding PNG. Let me do that now.

**Comment by artursouza** [comment]
Updated.

**Pull request merged**